reactor-core/src/main/java/reactor/core/publisher/MonoCacheTime.java

```diff
@@ -37,32 +37,38 @@
  *
  * @author Simon Baslé
  */
-class MonoCacheTime<T> extends MonoOperator<T, T> implements Runnable {
+class MonoCacheTime<T> extends MonoOperator<T, T> implements Scheduler.ContextRunnable {
```

Comment:
I'm not sure this particular class benefits from `ContextAware`. There's no actual `Context` to tie to, it's just `Runnable` for the purpose of expiration of the cache.